### ISSUE TITLE
Two non-related locations to set TLSA- Needs to be moved to server templates

### ISSUE THREAD

**Issue by shoulders** [virtualmin/virtualmin-gpl#701]
|  SYSTEM INFORMATION||
|----------------------|---------------------------|
|  OS type and version |     Ubuntu Linux 22.04.3     |
|  Webmin version      |     2.105      |
|  Usermin version     |     2.005     |
|  Virtualmin version  |     7.8.2       |
|  Theme version       |     21.09.5      |
|  Package updates     |      12  package updates are available   |

I referenced this in #696 but thought it might need its own issue

### The background
The TLSA option is in the following 2 locations
- Virtualmin --> System Settings --> Virtualmin Configuration --> Configuration category: SSL settings --> Create TLSA DNS records for SSL certificates
![image](https://github.com/virtualmin/virtualmin-gpl/assets/319997/4110875f-c9b4-4eef-b57c-c13b34a35277)

-  Virtualmin --> Server Configuration --> DNS Options --> TLSA records enabled?
![image](https://github.com/virtualmin/virtualmin-gpl/assets/319997/2a2e19d9-0996-4528-b324-73d49544de6b)

### The issue
As you can see there is an option in `Virtualmin Configuration` that says it will enable TLSA for all domains with an SSL and then there is a setting in each `virtual server (DNS Options)` to enable or disable the TLSA record. There are no settings in the Server templates to control TLSA.

The  `Virtualmin Configuration` setting is trying to behave like a `Server Template` option but instead of this option being configurable per domain,  when this setting is changed, all domains will have this setting changed.

Enabling TLSA is a per domain/virtual server option rather than a global Virtualmin configuration options.

### Proposed Solution
Moved the `Virtualmin Configuration` option into the `Server Templates` which is better fit for a per domain configurable option.

**Comment by jcameron** [comment]
Agreed, the TLSA option on the Virtualmin Configuration page should be in Server Templates. That way it controls the defaults for new domains, and the setting on DNS Options controls the current state for one domain.

**Comment by iliaross** [comment]
Reopened, as this was closed by an automated cleanup run. @shoulders, sorry for the noise, and thanks for the contribution. We definitely didn't mean any disrespect.

If it turns out the issue is already resolved for you, you're welcome to close it.
